src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,15 +27,23 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        nextSequenceNumber++;
+        sequenceNumber++;
```

Comment:
seems like all Persons will have sequence number 1? since sequenceNumber is an instance variable